library.properties

```
@@ -6,5 +6,5 @@ sentence=Arduino library for providing a convenient C++ interface for accessing
 paragraph=
 category=Communication
 url=https://github.com/107-systems/107-Arduino-UAVCAN
-architectures=*
+architectures=samd,mbed,esp32
```

```suggestion
architectures=samd,mbed,mbed_nano,mbed_portenta,esp32
```
As of the [2.0.0 release of ArduinoCore-mbed](https://github.com/arduino/ArduinoCore-mbed/releases/tag/2.0.0), the architecture name of the Nano 33 BLE was changed from `mbed` to `mbed_nano` and the architecture name of the Portenta H7 was changed from `mbed` to `mbed_portenta`. It is good to retain `mbed` to give backwards compatibility for people still using pre-2.0.0 versions of the platform.

---

Terrific catch! If that's all, please resolve your review and go ahead and merge this one.